sdk/spring/spring-integration-azure-eventhubs/src/main/java/com/azure/spring/integration/eventhubs/inbound/EventHubsInboundChannelAdapter.java

```diff
@@ -51,13 +51,20 @@
  *     }
  *
  *    {@literal @}Bean
- *     public EventHubsInboundChannelAdapter messageChannelAdapter(
- *             {@literal @}Qualifier("input") MessageChannel inputChannel,
- *             EventHubsMessageListenerContainer listenerContainer) {
+ *        public EventHubsInboundChannelAdapter messageChannelAdapter(
+ *         {@literal @}Qualifier("input") MessageChannel inputChannel, EventHubsProcessorFactory processorFactory) {
+ *         EventHubsContainerProperties containerProperties = new EventHubsContainerProperties();
+ *         containerProperties.setEventHubName(EVENTHUB_NAME);
+ *         containerProperties.setConsumerGroup(CONSUMER_GROUP);
```

Comment:
change the EVENTHUB_NAME and CONSUMER_GROUP to "eventhub-1" and "consumer-group-1"